src/command/FileCommands.cxx

```diff
@@ -330,3 +330,28 @@ handle_read_picture(Client &client, Request args, Response &r)
 	handler.RethrowError();
 	return CommandResult::OK;
 }
+
+class PrintLyricsHandler final : public NullTagHandler {
+	Response &response;
+
+public:
+	explicit PrintLyricsHandler(Response &_response) noexcept
+		:NullTagHandler(WANT_LYRICS), response(_response) {}
+
+	void OnLyrics(StringView value) noexcept override {
+		response.Format("%.*s\n",
+				int(value.size), value.data);
```

Comment:
So this writes the lyrics right into the stream, raw. How does the client know when the lyrics are finished?
What if there are multiple lyrics tags?